debug.log

```diff
@@ -0,0 +1,52 @@
+[0623/180920.289:ERROR:crash_report_database_win.cc(469)] failed to stat report
```

Comment:
You don't have to check in this file. You can add it to .gitignore file or just simply remove it.